codersdk/init_script.go

```diff
@@ -0,0 +1,28 @@
+package codersdk
```

Comment:
nit: This file should probably be renamed just `initscript.go` to match other files in the dir like `provisionerdaemons.go`. Files ending with `_xyz.go` are usually for showing stuff similar build tags in the filename